higher_health/templates/login.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+{% load temp_tags static compress cache %}
+{% block content %}
+  <div class="content">
+    <div class="content-intro">
+      <div class="content-inner">
+        <div class="heading-container">
+          <h1 class="heading heading-primary">Welcome to HealthCheck</h1>
+          <h2 class="heading heading-secondary">Your daily risk assessment tool.</h2>
+          <h3 class="heading heading-subtitle">Please help us by answering a few questions about you and your health.</h3>
+          <p>
+            You will receive advice on what actions to take based on national guidelines,
+            and the data you enter will help us in predictive modelling and planning
+            our national response to COVID-19.
+
+            Thank you for coming forward and for contributing to the
+            health of all citizens AND stopping the spread of the virus 🦠
+          </p>
+        </div>
```

Comment:
Not something for this PR, but we seem to be needing this block on a number of templates.

I'm wondering if this would be better as a fragment that we include, or many as part of the base template in a block that we can override.